services/paperless/docker/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Custom image combining restic with rclone for Paperless backups
+# This eliminates the need for runtime downloads and improves startup time and reliability
+
+# renovate: datasource=github-releases packageName=restic/restic versioning=semver
+ARG RESTIC_VERSION=0.17.3
+
+# renovate: datasource=github-releases packageName=rclone/rclone versioning=semver
+ARG RCLONE_VERSION=1.68.2
```

Comment:
The name of the `RCLONE_VERSION` env var conflicts with the `rclone version` check at the end of the dockerfile as rclone seems to try to evaluate it. Thus the env var should be renamed with some prefix to avoid rclone trying to use that.